pkg/controllers/service.go

```diff
@@ -68,7 +68,7 @@ func (r *ServiceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return ctrl.Result{}, nil
 	}
 
-	svcImport.Spec.IPs = []string{service.Spec.ClusterIP}
+	svcImport.Spec.IPs = service.Spec.ClusterIPs
```

Comment:
This may be problematic since the CRD limits the array to [one item](https://github.com/kubernetes-sigs/mcs-api/blob/eff1ba8c3ab2d50a7faa0a02d18dc6ef9f58a21d/pkg/apis/v1alpha1/serviceimport.go#L56).

Reply:
Ah yes indeed good point thanks! The alpha2 version on #52 does make this to two items max so I am thinking that we could just wait for this version somehow?

cc @mikemorris

Reply:
```suggestion
	svcImport.Spec.IPs = service.Spec.ClusterIPs[:2]
```
This field on ServiceImport is intended for parity with Service `ClusterIPs` (as noted in #63), which is [restricted to two items](https://github.com/kubernetes/kubernetes/blob/a499facee693a1a83daadb82d88f7b51d324ffc5/pkg/apis/core/types.go#L4688-L4696) but we still might want an extra safeguard here?

I do think this is blocked by #85 updating the max length to 2 as @tpantelis noted, but once that is in this LGTM.

Reply:
FTR you can't do this `service.Spec.ClusterIPs[:2]` if the the slice capacity is smaller than 2, I tweaked it to make this work by checking the len first though!